cicd/docker/algod/setup.py

```diff
@@ -15,32 +15,41 @@
 
 # Shared parameters
 base_parser = argparse.ArgumentParser(add_help=False)
-base_parser.add_argument('--bin-dir', required=True, help='Location to install algod binaries.')
-base_parser.add_argument('--channel', required=True, help='Channel to install, nightly is a good option.')
-base_parser.add_argument('--network-dir', required=True, help='Path to create network.')
 
 subparsers = parser.add_subparsers()
 
-install = subparsers.add_parser('install', parents=[base_parser], help='Install the network.')
-install.add_argument('--network-template', required=True, help='Path to private network template file.')
-install.add_argument('--network-token', required=True, help='Valid token to use for algod/kmd.')
-install.add_argument('--algod-port', required=True, help='Port to use for algod.')
-install.add_argument('--kmd-port', required=True, help='Port to use for kmd.')
+install = subparsers.add_parser('install', parents=[base_parser], help='Install binaries.')
+install.add_argument('--bin-dir', required=True, help='Location to install algod binaries.')
+install.add_argument('--channel', required=True, help='Channel to install, nightly is a good option.')
+install.add_argument('--algod-version', required=False, default='2.0.4', help='Channel to install, nightly is a good option.')
```

Comment:
This doesn't need to be configurable, the "update" script will always pull down the most recent version. If that changes then we'll need to update more than just this script.

Reply:
Oh I thought when we talked one of the things you wanted was to be able to run any of the versions of algod that we've built. I could change it, but the way it's written now this can run a container with any version of algod we've published running a network for testing/troubleshooting

Reply:
Ah you're right. Do you think it might be useful to be able to use a certain version rather than always running update? That might be convenient for testing/troubleshooting certain versions we've published

Reply:
A certain version of go-algorand? Maybe, I can imagine if there was a regression caused by an algod change, running the SDKs against an older version would verify that algod caused the regression.